geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/tools/PropertyManagement.java

```diff
@@ -68,46 +77,40 @@ public Object get(
 		return properties.get(toPropertyName(propertyName));
 	}
 
-	public void store(
+	public synchronized void store(
 			final ParameterEnum propertyName,
 			final Object value ) {
-		synchronized (MUTEX) {
-			properties.put(
-					toPropertyName(propertyName),
-					value);
-		}
+		properties.put(
```

Comment:
Then I will change it to a map.  The values need to serializable objects.  I will make the adjustment.